tests/test_simple_coverage_boost.py

```
@@ -1,47 +1,90 @@
+#!/usr/bin/env python3
```

Ruff (0.13.1)

Shebang is present but file is not executable

(EXE001)